src/components/Sponsors/Sponsors.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react'
+import { uid } from 'uid'
+import SponsorCard from '../SponsorCard/SponsorCard'
+import './Sponsors.css'
+
+const sponsors = [
+    {
+        name: "Wix Studio",
+        type: "Financial Sponsor",
+        description: "Lorem ipsum dolor sit amet, consectetur adipisicing elit. ",
+        logo: "https://static.wixstatic.com/media/0d6674_b39c8ddaafbd4b8dbb459a4221ee8227~mv2.png/v1/fill/w_524,h_519,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/Frame%2010000017556.png", 
```

Comment:
We likely want to use the .png from /assets, but that can be fixed later when the filler gets replaced with real copy.